Sources/PeakCoreData/Operations/CoreDataBatchDeleteOperation.swift

```diff
@@ -14,7 +14,10 @@ open class CoreDataBatchDeleteOperation<Entity: ManagedObjectType>: CoreDataOper
     private let predicate: NSPredicate?
     private let mergeContexts: [NSManagedObjectContext]?
 
-    public init(predicate: NSPredicate? = nil, mergeContexts: [NSManagedObjectContext]? = nil, persistentContainer: NSPersistentContainer, mergePolicyType: NSMergePolicyType = .mergeByPropertyObjectTrumpMergePolicyType) {
+    public init(predicate: NSPredicate? = nil,
+                mergeContexts: [NSManagedObjectContext]? = nil,
+                persistentContainer: NSPersistentContainer,
```

Comment:
`persistentContainer` is the one required parameter but it sits in the middle of the parameter list. Is there a better order?

Reply:
I think the rationale here was that the properties that are needed for the parent class go at the end. So because this subclasses `CoreDataOperation` the persistentContainer and mergePolicyType go at the end, and the two properties needed for this class go in front.